step-core/src/main/java/step/core/entities/EntityManager.java

```diff
@@ -91,9 +94,10 @@ public void getEntitiesReferences(String entityType, ObjectPredicate objectPredi
 		if (entity == null ) {
 			throw new RuntimeException("Entity of type " + entityType + " is not supported");
 		}
-		entity.getAccessor().getAll().forEachRemaining(a -> {
+		Filter exportAllFilters = Objects.requireNonNullElse(getExportAllFilters(entityType), Filters.empty());
+		entity.getAccessor().getCollectionDriver().find(exportAllFilters,null,null,null,0).forEach(a -> {
```

Comment:
This could really benefit from a short comment explaining what it does and why (i.e. what do the conditions mean)

Reply:
added comments